inc/html.class.php

```diff
@@ -959,15 +949,12 @@ static function getConfirmationOnActionScript($string, $additionalactions = '')
     *
     * @since 0.85
     *
-    * @param $id                 HTML ID of the progress bar
-    * @param $options    array   progress status
-    *                    - create    do we have to create it ?
-    *                    - message   add or change the message
-    *                    - percent   current level
-    *
-    *
-    * @return nothing (display)
-    **/
+    * @param integer $id        HTML ID of the progress bar
+    * @param array   $options   progress status
+    *                           - create    do we have to create it ?
+    *                           - message   add or change the message
+    *                           - percent   current level
+    */
```

Comment:
It returns void.

Reply:
I only fix the arguments here but actually in some cases this prints html so you are returning a string, does it count?